### DISCUSSION TITLE
Layer Blend Modes

### DISCUSSION THREAD

**Discussion by jasoncmcg** [Orama-Interactive/Pixelorama#1021] [Ideas]
Layer blend mode mainly for color effects, like Gimp or Photoshop.
Enable non-destructive type edits for things like shadow/highlights.

**Comment by Variable-ind** [comment]
Hi 👋, Layer blend modes and FX, and clip masks (and more) are now part of the upcoming version 1.0 🎉

(More on blend modes [Here](https://github.com/Orama-Interactive/Pixelorama/pull/911))
![275332936-b5201842-c1ad-43ef-8e07-2fb917717955](https://github.com/Orama-Interactive/Pixelorama/assets/77773850/41f9e481-2862-4e13-95fd-0c4eaa9e11b8)

(More on non destructive layer effects [Here](https://github.com/Orama-Interactive/Pixelorama/pull/940))

https://github.com/Orama-Interactive/Pixelorama/assets/77773850/da8d0e23-4c74-4b1f-8176-8254dca12851

**Reply by jasoncmcg** [reply]
I can't wait, that looks better than what I hoped for!
